src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../../types/tab';
+
+type FormProps = {
+  tabs: Tab[]
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<FormProps> = (
+  { tabs, selectedTabId, onTabSelected },
+) => {
+  const selectedTab = tabs.find(i => i.id === selectedTabId) || tabs[0];
+
+  const handleClick = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+    }
```

Comment:
I like to avoid nesting as much as possible ;)
```suggestion
    if (tab.id === selectedTabId) return;
    
     onTabSelected(tab);
```